### PR TITLE
cosmetic changes

### PR DESCRIPTION
**Pull request by vchvlad**

### CODE REVIEW

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts [hunk 1]
```diff
@@ -96,8 +95,14 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 		];
 	}
 
-	private async createToolbar(view: azdata.ModelView): Promise<azdata.ToolbarContainer> {
+	private limitLongName(name: string): string {
+		if (name.length > 40) {
```
Comment (Charles-Gagnon) [inline extensions/sql-assessment/src/tabs/assessmentMainTab.ts:99]: Normally for this you'd just use the text overflow. Manually changing the string like this isn't great for screen readers.

Although I'm wondering where this is even necessary - the only strings that use this are all < 40 characters.

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts [hunk 2]
```diff
@@ -107,18 +112,20 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 				light: this.extensionContext.asAbsolutePath('resources/light/database.svg')
 			};
 		const iconSize: number = 16;
+		const btnHeight: string = '26px';
 
 		const btnInvokeAssessment = view.modelBuilder.button()
 			.withProperties<azdata.ButtonProperties>({
-				label: this.invokeAssessmentLabel,
+				label: this.limitLongName(this.invokeAssessmentLabel),
```
Comment (Charles-Gagnon) [inline extensions/sql-assessment/src/tabs/assessmentMainTab.ts:119]: Why is this necessary? This string will never be over 40 chars

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts [hunk 3]
```diff
@@ -96,8 +95,14 @@ export class SqlAssessmentMainTab extends SqlAssessmentTab {
 		];
 	}
 
-	private async createToolbar(view: azdata.ModelView): Promise<azdata.ToolbarContainer> {
+	private limitLongName(name: string): string {
```
Comment (Charles-Gagnon) [inline extensions/sql-assessment/src/tabs/assessmentMainTab.ts:98]: Side note - but functions like this generally are better put as standalone functions at the module level instead of in a class since they don't actually need any class members.

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts (hunk 2 above)
Comment (vchvlad) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts:119]: we need to handle cases for very-very long database names and small screens (like 13 inch). The label is composed of "Invoke assessment for <databasename>", so limit it by length seems reasonable to solve the issue.

Comment (Charles-Gagnon) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts:119]: Oh I didn't see you overwrote the title if it's a server connection....could you attach an image of what this looks like?

Comment (Charles-Gagnon) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts:119]: (for future reference please always attach screenshots/gifs if you're making UI changes so reviewers can easily see what the changes look like)

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts (hunk 1 above)
Comment (vchvlad) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts:99]: I spent a lot of time but wasn't able to accomplish this by css styles, my custom styles were not applied to toolbar item.

**Comment by vchvlad** [comment]
Solving an issue with long names. Without fix it looks like this: 
image

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts (hunk 2 above)
Comment (vchvlad) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts:119]: I have attached the screen with an issue at the level up comments.

Comment (Charles-Gagnon) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts]: Why do you even need the database name here? The dashboard already indicates which database it is so my opinion is just that you should remove the name completely.

File: extensions/sql-assessment/src/tabs/assessmentMainTab.ts (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/sql-assessment/src/tabs/assessmentMainTab.ts]: Hm, ok. Might be an issue with the core but not something I'm going to be able to investigate right now so this is fine.
